app/controllers/api/v1/pizzas_controller.rb

```diff
@@ -1,8 +1,29 @@
 class Api::V1::PizzasController < ApplicationController
+  protect_from_forgery unless: -> { request.format.json? }
 
   def show
     selected_pizza = Pizza.find(params[:id])
 
     render json: selected_pizza, serializer: PizzaSerializer
   end
+
+  def create
+    pizza = Pizza.new(pizza_params)
+    brand = pizza.brand
+
+    if pizza.save 
+      errors = "Pizza added succesfully."
+      render json: {pizzas: brand.pizzas, errors: [errors]}
```

Comment:
I wouldn't consider a success message an error, you could define this as `message` if you wanted to, but I don't think a success message is required